static/js/narrow_state.js

```diff
@@ -89,7 +89,16 @@ exports.set_compose_defaults = function () {
     }
 
     if (single.has('pm-with')) {
-        opts.private_message_recipient = single.get('pm-with');
+        var private_message_recipient = single.get('pm-with');
+        var valid_op = false;
+        private_message_recipient.split(',').forEach(function (op) {
+            if (people.get_by_email(op)) {
+                valid_op = true;
+            }
+        });
+        if (valid_op) {
+            opts.private_message_recipient = private_message_recipient;
+        }
```

Comment:
I think the `valid_op` logic is wrong -- we should require all recipients to be valid.

Reply:
I implemented it this way so that even if one email is valid, it will show the compose box and won't show the non existent user message as there is still one valid email.
It can be implemented the way that all recipients should be valid. Then how will we handle the case when some of the operands are valid and other invalid?